HousingSearchApi.Tests/V1/E2ETests/Steps/GetAssetSteps.cs

```diff
@@ -107,15 +107,35 @@ public async Task WhenAnExactMatchExists(string address)
 
             _lastResponse = await _httpClient.GetAsync(route).ConfigureAwait(false);
         }
-
-
         public async Task WhenAnExactMatchExistsAndIsFilteredQueryTrue(string address)
         {
             var route = new Uri($"api/v1/search/assets?searchText={address}&isFilteredQuery=true&pageSize={1}",
                 UriKind.Relative);
 
             _lastResponse = await _httpClient.GetAsync(route).ConfigureAwait(false);
         }
+        public async Task WhenIsTemporaryAccomodation(string isTemporaryAccommodation)
+        {
+            var route = new Uri($"api/v1/search/assets/all?isTemporaryAccomodation={isTemporaryAccommodation}&page={1}",
+                UriKind.Relative);
+
+            _lastResponse = await _httpClient.GetAsync(route).ConfigureAwait(false);
+        }
+        public async Task WhenIsTemporaryAccomodationAndSearchText(string searchText)
+        {
+            var route = new Uri($"api/v1/search/assets/all?isTemporaryAccomodation=true&searchText={searchText}&page={1}",
+                UriKind.Relative);
+
+            _lastResponse = await _httpClient.GetAsync(route).ConfigureAwait(false);
+        }
+        public async Task ThenOnlyTemporaryAccomodationResultsShouldBeIncluded(bool isTemporaryAccommodation)
```

Comment:
Is there a conflict between the test name and the input paramater handling since the boolean value will determine what values wil be returned?

Reply:
Yes - have updated that so it's always true and the name of the When task 'WhenIsTemporaryAccomodationIsTrueAndSearchText'